src/components/CheckEditor/checkFormTransformations.ts

```diff
@@ -183,11 +185,26 @@ const getHttpSettingsFormValues = (settings: Settings): HttpSettingsFormValues =
   };
 };
 
+const getTcpQueryResponseFormValues = (queryResponses?: TCPQueryResponse[]) => {
+  if (!queryResponses) {
+    return undefined;
+  }
+  return queryResponses.map(({ send, expect, startTLS }) => {
+    return {
+      startTLS,
+      send: atob(send),
+      expect: atob(expect),
```

Comment:
can both `btoa()` and `atob()` methods be in a try/catch block? If there is an invalid character it throws an exception

Reply:
Good call!